### ISSUE TITLE
Customize or disable serve console message

### ISSUE THREAD

**Issue by csicky** [vuejs/vue-cli#3726]
### What problem does this feature solve?
I would like to be able to see only if there is a problem, if it's green, yellow or red. I don't want to give so much screen to the console, but in order to see what I want I have to see also the entire message which I know already:

```
 DONE  Compiled successfully in 24499ms      
  App running at:
  - Local:   http://localhost:8080/
  - Network: http://192.168.100.10:8080/

  Note that the development build is not optimized.
  To create a production build, run npm run build.
```

### What does the proposed API look like?
I need a way to have the console say only:

```
 DONE  Compiled successfully in 24499ms  
```

### Not duplicate of https://github.com/vuejs/vue-cli/issues/2115, that issue was a different topic.

**Comment by LinusBorg** [comment]
...but that message only appears on first compile?

**Comment by csicky** [comment]
Yes, the "development build not optimized" message appears only once, but still the "app running at" is taking up vertical space. If I try to make the console height to take only one line, I don't see anymore if it's green red or yellow. I actually only care about the colors.

![image](https://user-images.githubusercontent.com/3931907/55241979-f862f780-5244-11e9-97d3-af879b83711d.png)


What I do to is I go to serve.js in `node_modules` folder and comment out the noisy `console.log` calls. This is problematic when I update packages because I have to go and do it again.

Thank you for your help.
